### ISSUE TITLE
Documentation for efficient storage of models

### ISSUE THREAD

**Issue by karthikv2k** [xmartlabs/Bender#75]
"Bender allows you to use efficient storage methods to keep the models up to 10x smaller in size. "
The bender website has this statement but I couldn't any documentation on this.

**Comment by joaquinrok** [comment]
Hey Karthik,
The initial model storage used by CoreML was really inefficient but that seems to have changed by now and the Bender website has been updated to reflect that. Our latest tests indicate CoreML models to be about the same size of having binary weights so no need to worry anymore. If you had had the chance to create CoreML models and compare initial binary weights storage size to the generated model let us know, we are always researching.
On another note, we did discover that CoreML limits the raw performance when compared to the MPS shaders, up to 40% in some cases so that may be a new reason to give it a shot! Take it with a grain of salt for now, we are still digging into this matter and plan to release further data in the near future, sit tight!

I'm closing the issue given that the website was dated and has been updated by now.
Cheers!
